CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# 107.1.0
+
+This minor release we added support to Snackbar to inform users when it has disappeared. As well as, improvements around touch targets for Chips, alignment improvements to Dialogs and various other bug improvements.
```

Comment:
This minor release includes additions to the `MDCSnackbarManagerDelegate` protocol, touch target improvements to Chips, alignment improvements to Dialogs, and various other bug fixes.

Reply:
Done